backend/composer/services/cs_ingestion/helpers/statement_helper.py

```diff
@@ -35,11 +35,11 @@ def create_or_update_connectivity_statement(statement: Dict, sentence: Sentence,
         reference_uri=reference_uri,
         defaults=defaults
     )
-    if not created:
+    if not created:        
         if has_changes(connectivity_statement, statement, defaults):
-            ConnectivityStatement.objects.filter(reference_uri=reference_uri).update(**defaults)
-            fields_to_refresh = [field for field in defaults.keys() if field != 'state']
-            connectivity_statement.refresh_from_db(fields=fields_to_refresh)
+            defaults_without_state = {field: value for field, value in defaults.items() if field != 'state'}
+            ConnectivityStatement.objects.filter(reference_uri=reference_uri).update(**defaults_without_state)
```

Comment:
@afonsobspinto - keeping the filter instead of the get here - since we cannot update a get
`AttributeError: 'ConnectivityStatement' object has no attribute 'update'`

also doing `connectivity_statement = ConnectivityStatement.objects.filter(reference_uri=reference_uri).first()` since, we get 

```
ConnectivityStatement.objects.filter(reference_uri=reference_uri).update(**defaults_without_state)
>>> 1
```
as return value after the update.

Reply:
I believe we could do:

            ```
            ConnectivityStatement.objects.filter(reference_uri=reference_uri).update(**defaults_without_state)
            connectivity_statement.refresh_from_db()
            ```

Reply:
Do you mean the following?

```
fields_to_refresh = [field for field in defaults.keys() if field != 'state']
connectivity_statement.refresh_from_db(fields=fields_to_refresh)
```


since the `connectivity_statement.refresh_from_db()` would lead to `AttributeError: Direct state modification is not allowed`.